File: Programming-languages-and-laboratory/Java/Exercises/08_compito_Java 2016-02-16/Privato.java
import java.util.*;

public class Privato extends Transazione {
	
  private String nome;
  private String indirizzo;
  private String codiceFiscale;

  
  public Privato(int id, int giorno,
  int mese,int anno,String nome, String indirizzo, String codiceFiscale){
    super(id,giorno,mese,anno);
    this.nome = nome;
    this.indirizzo = indirizzo;
    this.codiceFiscale = codiceFiscale;
  }
  
  public String toString(){
	  return id + "\t"+giorno+"/"+mese+"/"+anno+"\t" + nome + "\t" + indirizzo + "\t" + 
    codiceFiscale +"-\t-\t-";
  }
}

File: Programming-languages-and-laboratory/Python/Exercises/08_compito_Java 20160216/Privato.py
from Acquisto import Acquisto


class Privato(Acquisto):
    def __init__(self, tipo, codice, giorno, mese, anno, nome_cognome, indirizzo, codice_fiscale):
        super().__init__(tipo, codice, giorno, mese, anno)
        self._nome_cognome = nome_cognome
        self._indirizzo = indirizzo
        self._codice_fiscale = codice_fiscale

    def __str__(self):
        return super().__str__() + "\t" + self._indirizzo + "\t" + self._nome_cognome + "\t" + self._codice_fiscale + "\t-\t-\t-"
